### ISSUE TITLE
404 errors with credential manager

### ISSUE THREAD

**Issue by johnwc** [elsa-workflows/elsa-core#3647]
Following the [directions for the UI](https://github.com/elsa-workflows/elsa-core/tree/master/src/modules/secrets/Elsa.Secrets) for the secrets module, aka Credentials Manager, I received the following error in the browser.

<img width="1061" alt="image" src="https://user-images.githubusercontent.com/2798441/214129178-a9ad9d7d-38c8-43f7-b6b7-e27883a59c75.png">
 When I change it like the following, it works.

```javascript
// From...
import { CredentialManagerPlugin } from "/build/index.esm.js";
// To...
import { CredentialManagerPlugin } from "@basePath/_content/Elsa.Designer.Components.Web/elsa-workflows-studio/index.esm.js";
```
   
   

When I update it to the correct path and click the `Credential Manager` link at the top of the page, I received the following error in the browser and nothing loads.

<img width="1055" alt="image" src="https://user-images.githubusercontent.com/2798441/214130017-7035d492-328a-4a5a-99e8-293b83f6579e.png">

**Comment by johnwc** [comment]
@sfmskywalker can anyone look at this? This is stopping us from being able to use credential manager.

**Comment by sfmskywalker** [comment]
Hi @johnwc , I'm a bit swamped at the moment, but I'll ask the author to see if they might be able to help short-term. @tanelkuhi @ke-real any thoughts on this one?

**Comment by johnwc** [comment]
@tanelkuhi @ke-real any thoughts?

**Comment by johnwc** [comment]
@sfmskywalker I think this is now fixed via #3739
